static/js/download.js

```diff
@@ -4,24 +4,35 @@
   var os = n.platform.match(/(Win|Mac|Linux)/);
   var x = n.userAgent.match(/x86_64|Win64|WOW64/) ||
     n.cpuClass === 'x64' ? 'x64' : 'x86';
+  var text = 'textContent' in d ? 'textContent' : 'innerText';
   var db = d.getElementById('home-downloadbutton');
-  if (!db) { return; }
-  var version = db.dataset.version;
-  var dlLocal = db.dataset.dlLocal;
-  var text = 'textContent' in db ? 'textContent' : 'innerText';
-  switch (os && os[1]) {
-    case 'Mac':
-      db.href += 'node-' + version + '.pkg';
-      db[text] = dlLocal + ' OS X (x64)';
-      break;
-    case 'Win':
-      // Windows 64-bit files for 0.x.x need to be prefixed with 'x64/'
-      db.href += (version[1] == '0' && x == 'x64' ? x + '/' : '') + 'node-' + version + '-' + x + '.msi';
-      db[text] = dlLocal + ' Windows (' + x +')';
-      break;
-    case 'Linux':
-      db.href += 'node-' + version + '-linux-' + x + '.tar.gz';
-      db[text] = dlLocal + ' Linux (' + x + ')';
-      break;
+  var version;
+  if (db) {
+    version = db.dataset.version;
+    var dlLocal = db.dataset.dlLocal;
```

Comment:
Still alot of users coming from < IE 11. Just verified locally that this doesnt work for IE 9. Maybe you could fix this issue in this PR aswell?

Using `db.getAttribute('data-version')` and `db.getAttribute('data-dl-local')` respectively does the trick.